new/LIT5848MagicPrayer.xml

```diff
@@ -0,0 +1,56 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5848MagicPrayer" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Magic Prayer of Devil</title>
```

Comment:
I'm sorry, I have trouble understanding the English - is it a prayer of the Devil (he is praying)? or a prayer against him?

Reply:
Maybe it is "Kehdata Saytan" a prayer addressing the Devil?

Reply:
@thea-m the description in the catalogue is not clear as well, but it is definitely not a prayer against devil. @DenisNosnitsin1970 I think it might be "Kehdata Saytan", but the information in the catalogue is not sufficient.

Reply:
Interesting! Maybe a note would be useful, depending on how high you estimate the probability to be? In any case, I think the devil should have an article, it sounds strange to me now - "Magic prayer of the devil"?